components/tabs/w-tab.vue

```diff
@@ -22,12 +22,7 @@ onBeforeUnmount(() => {
   controller?.unregisterTab?.(props.name);
 });
 
-const tabClasses = computed(() => ({
-  [ccTab.tab]: true,
-  [ccTab.tabInactive]: !isActive.value,
-  [ccTab.tabActive]: isActive.value,
-  ['active-tab']: isActive.value,
-}));
+const tabClasses = computed(() => [ccTab.base, isActive.value && 'active-tab', isActive.value ? ccTab.active : ccTab.inactive]);
```

Comment:
Not a big deal, but perhaps we could skip one `isActive.value` check and do something like this:
```suggestion
const tabClasses = computed(() => [ccTab.base, isActive.value ? `active-tab ${ccTab.active}` : ccTab.inactive]);
```

Reply:
Updated in this commit: [8674cd8](https://github.com/warp-ds/vue/pull/187/commits/8674cd8ad1aa706a910827577ca174c06fe20e72)